pkg/controller/nodemaintenance/nodemaintenance_controller.go

```diff
@@ -206,15 +232,12 @@ func (r *ReconcileNodeMaintenance) Reconcile(request reconcile.Request) (reconci
 		return r.reconcileAndError(instance, err)
 	}
 
-	if err = runCordonOrUncordon(r, node, true); err != nil {
+	if err = drain.RunCordonOrUncordon(r.drainer, node, true); err != nil {
 		return r.reconcileAndError(instance, err)
 	}
 
-	stop := make(chan struct{})
-	defer close(stop)
-
-	reqLogger.Infof("Evict all Pods from Node: %s", nodeName)
-	if err = drainPods(r, node, stop); err != nil {
+	err = drain.RunNodeDrain(r.drainer, nodeName)
+	if err != nil {
```

Comment:
Lets follow the existing convention...
```suggestion
	if err = drain.RunNodeDrain(r.drainer, nodeName); err != nil {
```

Reply:
force pushed as part of the last commit.